content/webapp/services/prismic/api.ts

```diff
@@ -0,0 +1,40 @@
+import * as prismic from 'prismic-client-beta';
+import { PrismicDocument } from '@prismicio/types';
+import { GetServerSidePropsContext } from 'next';
+import fetch from 'node-fetch';
+
+type Req = GetServerSidePropsContext['req'];
+
+const routes = [
+  {
+    type: 'seasons',
+    path: '/seasons/:uid',
+  },
+];
+const endpoint = prismic.getEndpoint('wellcomecollection');
+const client = prismic.createClient(endpoint, { routes, fetch });
+
+// We enforce have a req object as we should never be using the API on the client.
+// To fetch stuff via the client, use the ./api paths.
+const initialised = false;
```

Comment:
when does this become true?

Reply:
Forgot to do that, it's done now. Maybe some tests here to explain usage would be better.

Reply:
[Tests are added](https://github.com/wellcomecollection/wellcomecollection.org/pull/7321/commits/0405f1d15428737f2b5a6b3b6e703559c2b3509b).